src/main/java/wolox/training/config/swaggerConfiguration.java

```diff
@@ -0,0 +1,38 @@
+package wolox.training.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.service.ApiInfo;
+import springfox.documentation.service.Contact;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+import springfox.documentation.swagger2.annotations.EnableSwagger2;
+
+@Configuration
+@EnableSwagger2
+public class swaggerConfiguration {
+
+    @Bean
+    public Docket productApi() {
+        return new Docket(DocumentationType.SWAGGER_2).select()
+            .apis(RequestHandlerSelectors.any())
+            .paths(PathSelectors.any())
+            .build().apiInfo(apiEndPointInfo());
+    }
+
+    private ApiInfo apiEndPointInfo(){
+        return new ApiInfoBuilder().title("Wolox Training")
+            .description("REST API for libraries")
```

Comment:
I recommend changing the default swagger message in .description (). You could put a description of what your application does.